pylint/config/__init__.py

```diff
@@ -77,12 +79,12 @@
             )
 
 
-def _get_pdata_path(base_name, recurs):
+def _get_pdata_path(base_name: str, recurs: int) -> pathlib.Path:
     base_name = base_name.replace(os.sep, "_")
-    return os.path.join(PYLINT_HOME, f"{base_name}{recurs}.stats")
+    return pathlib.Path(PYLINT_HOME) / f"{base_name}{recurs}.stats"
```

Comment:
We changed the typing here, right ?

Reply:
Yes, but it is a private function only used twice in this file. I thought I would conform to your `Path-love` 😄